azurerm/resource_arm_virtual_network_gateway.go

```diff
@@ -91,6 +91,9 @@ func resourceArmVirtualNetworkGateway() *schema.Resource {
 					string(network.VirtualNetworkGatewaySkuNameErGw1AZ),
 					string(network.VirtualNetworkGatewaySkuNameErGw2AZ),
 					string(network.VirtualNetworkGatewaySkuNameErGw3AZ),
+					string(network.VirtualNetworkGatewaySkuNameVpnGw1AZ),
+					string(network.VirtualNetworkGatewaySkuNameVpnGw2AZ),
+					string(network.VirtualNetworkGatewaySkuNameVpnGw3AZ),
```

Comment:
could we [update the documentation to include these new values](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/virtual_network_gateway.html.markdown#L138)?